script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -1,261 +1,285 @@
+ There is nothing to be gained from such an immortality.
 }
 [sha:a717118733c11f446bd940651e963b9c44cbe77a]{
 -- Page 6, Offset 37025.
 -- 私は、弱者であるが故に変わり続ける人間として、その純度を保ったまま在り続けねばならない。
-I'm weak, and therefore, as a human who continues to change, I must continue to hold on to that purity.
+As a human who is constantly driven to change by virtue of their own weakness,
+ I have no choice but to continue to cling to that purity.
 }
 [sha:f89ec143794546093c03e406c5e6eac039e7b0f1]{
 -- Page 7, Offset 37026.
 -- 何の為に？
-But why?
+Why must I do so?
 }
 [sha:c3d23e20b829d8af6448e30363301dd49c9c28bd]{
 -- Page 7, Offset 37027.
 -- 言うまでもない。そうでなければ意味がないからだ。
-It goes without saying. Because if I don't, then there would be no meaning.
+That goes without saying.
+ Without it, nothing would have any meaning.
 }
 [sha:f721ac9e4c50b780770785c07199933bddf4b507]{
 -- Page 7, Offset 37028.
 -- 私は観測者としてではなく当事者として、
-I must know the final destination of humanity,
+// Flopped with below for flow
+I must know the final destination of humanity.
 }
 [sha:884217d0972b867f3e97f050a23ca7869f117577]{
 -- Page 7, Offset 37029.
 -- 人間の到達地点を知らねばならない。
-not as an observer, but as a participant.
+Not as a mere observer, but as a participant.
 }
 [sha:a799d221061e88420455f277d8c726d8c431c501]{
 -- Page 7, Offset 37030.
 -- そうでなくては―――あの女には届くまい。
-Because if I don't―――I'll never be able to reach that woman.
+Because if I don't――I'll never be able to reach that woman.
 }
 [sha:c797f724cb474dabf00bfb965511390e3a73dc8b]{
 -- Page 8, Offset 37031.
 -- ……やはり、転生という手段をとろう。
-...I should just use reincarnation as a method.
+I suppose reincarnation is my best option after all...
 }
 [sha:6c72b7bbbd2345f222f417604c5eb735504fe551]{
 -- Page 8, Offset 37032.
 -- 私は一から生まれ直し、私となってまた死ねばいい。
-I can be born again from square one, and I can simply die again once I turn into myself.
+I can be born again from square one,
+ grow into myself,
+ and die once more.
 }
 [sha:e4ad310f8778482f982dbcbd8e378e383bdaad37]{
 -- Page 8, Offset 37033.
 -- この循環の中でなら、私は私という純度を保ったまま<在|い>きられる筈だ。
-If I stay within that cycle, I should be able to continue living while keeping my own purity intact.
+So long as I remain within that cycle,
+ I should be able to keep on existing while preserving my own purity.
 }
 [sha:6ddde8e7a13e27996823546cdd20131aa6a55361]{
 -- Page 9, Offset 37034.
 -- 個人としての永続に執着するのではなく、
-Instead of being fixated on permanence as an individual,
+I will not cling to the concept of permanence as an individual―――
 }
 [sha:c14c11b2c1d2f849bd4892ef2b2014664d359191]{
 -- Page 9, Offset 37035.
 -- 結果としての永続に漂流し続ける。
-I would continuously drift into permanence as a result.
+// Reading 結果 here as like 'permanent as a consequence of the fact that he continues to reincarnate',
+// rather than like a singular point _result_, hence rendering it as 'side effect' here even thoughts that's not
+// really the dictionary def
+Instead, I will continue to drift toward permanence as a side effect.
```

Comment:
Ehh, I don't think this is too handholdy - 個人 is a lot clearer than 'individual' (a single permanence?).

Reply:
Those mean the same thing

Reply:
What I'm trying to say is that 個人 always refers to an individual human (I assume?) but 'individual' in EN is too easy to misread here as a single version / instance / form of permanence, instead of permanence _as an individual_ - I would misread this sentence in EN

Reply:
I still think 'misreading' that doesn't matter, but I guess 'personal' would be less liable to that kind of reading.